connector/src/main/java/org/geysermc/connector/utils/LoginEncryptionUtils.java

```diff
@@ -152,43 +155,60 @@ private static void startEncryptionHandshake(GeyserSession session, PublicKey ke
         session.getUpstream().sendPacketImmediately(packet);
     }
 
-    private static int AUTH_FORM_ID = 1337;
+    private static int AUTH_FORM_ID = 1336;
+    private static int AUTH_DETAILS_FORM_ID = 1337;
 
     public static void showLoginWindow(GeyserSession session) {
-        CustomFormWindow window = new CustomFormBuilder("Login")
-                .addComponent(new LabelComponent("Minecraft: Java Edition account authentication."))
+        SimpleFormWindow window = new SimpleFormWindow("Login", "Minecraft: Java Edition account authentication. You need a Java Edition account to play on this server.");
```

Comment:
Maybe worth just having `You need a Java Edition account to play on this server.` and removing `Minecraft: Java Edition account authentication.` just to clean up the text a bit.